### PR TITLE
docs: add English version of standalown_vs_cluster

### LINKED ISSUE #1490: doc translation (Chinese to English): standalone_vs_cluster
- Translate the Chinese version to English: https://openmldb.ai/docs/zh/v0.4/tutorial/standalone_vs_cluster.html (you can download as a markdown file from the button at the top-right corner)
- Save the English version to docs/en/standalone_vs_cluster.md

### PR DESCRIPTION
**Pull request by cswjd**
docs: add English version of standalown_vs_cluster
close #1490

### CODE REVIEW

**Comment by cswjd** [comment]
fixes #1490

File: docs/en/standalone_vs_cluster.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Difference between OpenMLDB cluster version and standalone version
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: capitalize the first letter of all words 
(for titles, we should do this, please also help revise other titles throughout the doc)

Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: capitalize the first letter of all words 
(for titles, we should do this)

File: docs/en/standalone_vs_cluster.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## 1. The difference between installation and deployment
+
+The cluster version and the standalone version have their own deployment methods. For details, see [Installation and Deployment Details](../deploy/install_deploy.md). All in all, the main differences are:
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: All in all => In summary

Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: All in all => In summary

File: docs/en/standalone_vs_cluster.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+- The cluster version needs to install and deploy zookeeper
+- The cluster version needs to install task-manager
+
+## 2. Differences in usage
+
+### 2.1 Different workflow
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: Different Workflows

Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: Different Workflows

File: docs/en/standalone_vs_cluster.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+| Cluster Edition Workflow | Standalone Edition Workflow | Difference Description |
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: Cluster Version's Workflow
Standalone Version's Workflow
Difference

Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: Cluster Version's Workflow
Standalone Version's Workflow
Difference

File: docs/en/standalone_vs_cluster.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+| ---------------- | ---------------- | --------------- --------------------------------------------- |
+| Create Database and Tables | Create Database and Tables | None |
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: Create databases and tables

Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: Create databases and tables

File: docs/en/standalone_vs_cluster.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+| Offline data preparation | Data preparation | The cluster version of OpenMLDB needs to prepare offline data and online data separately. <br />The stand-alone version can use the same data or prepare different data for offline and online feature calculation. |
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: stand-alone => standalone (keep it consistent)

Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: stand-alone => standalone (keep it consistent)

Comment (lumianph) [reply docs/en/standalone_vs_cluster.md]: feature calculation => feature extraction

File: docs/en/standalone_vs_cluster.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+| Offline Feature Computation | Offline Feature Computation | None |
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: Offline feature extraction

File: docs/en/standalone_vs_cluster.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+| SQL Plan Online | SQL Plan Online | None |
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: SQL Plan Online => SQL deployment

File: docs/en/standalone_vs_cluster.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+| Online data preparation | None | The cluster version of OpenMLDB needs to prepare offline data and online data separately. <br />The stand-alone version can use the same data or prepare different data for offline and online feature calculation. |
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: feature calculation => feature extraction

File: docs/en/standalone_vs_cluster.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+| Online real-time feature calculation | Online real-time feature calculation | None |
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: feature calculation => feature extraction

File: docs/en/standalone_vs_cluster.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+
+
+
+### 2.2 Execution mode
+
+The cluster version supports the system variable `execute_mode`, which supports configuring the execution mode.
+
+When executed on the cluster version command line:
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: You can execute the below command in CLI to set the execution mode.

File: docs/en/standalone_vs_cluster.md [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+
+```sql
+> SET @@execute_mode = "offline"
+````
+
+OpenMLDB switches to offline execution mode. In this mode, only offline data will be imported/inserted and queried.
+
+When executed on the cluster version command line:
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: remove this sentence

File: docs/en/standalone_vs_cluster.md [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+> SET @@execute_mode = "online"
+````
+
+OpenMLDB switches to online execution mode. In this mode, only online data will be imported/inserted and queried.
+
+### 2.3 Offline task management
+
+Offline task management is a unique feature of the cluster version.
+
+The standalone version `LOAD DATA` data, `SELECT INTO` command is blocking, the cluster version will submit a task, and provide `SHOW JOBS`, `SHOW JOB` commands to view offline tasks. For details, see [Offline Task Management](../reference/sql/task_manage/reference.md).
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: The `LOAD DATA` and `SELECT INTO` command are blocking in the standalone version. However, the cluster version submits a task for those commands, and provides the commands `SHOW JOBS` and `SHOW JOB` to investigate the status of offline tasks. For details, see [Offline Task Management](../reference/sql/task_manage/reference.md).

File: docs/en/standalone_vs_cluster.md [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+
+### 2.4 SQL Boundaries
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: SQL Functionalities

File: docs/en/standalone_vs_cluster.md [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+
+The differences in SQL query capabilities supported by the cluster version and the standalone version include:
+
+- [Offline task management statement](../reference/sql/task_manage/reference.md)
+  - Standalone version of OpenMLDB does not support
+  - The cluster version of OpenMLDB supports offline task management statements, including: `SHOW JOBS`, `SHOW JOB`, etc.
+- Execution mode
+  - Standalone version of OpenMLDB does not support
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: The standalone version of OpenMLDB does not support setting execution mode.

File: docs/en/standalone_vs_cluster.md [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+  - Clustered OpenMLDB can configure the execution mode: `SET @@execute_mode = ...`
+- Use of `CREAT TABLE`[create table statement](../reference/sql/ddl/CREATE_TABLE_STATEMENT.md)
+  - The standalone version of OpenMLDB does not support configuring distributed properties
+  - Cluster version of OpenMLDB supports configuring distributed properties: including `REPLICANUM`, `DISTRIBUTION`, `PARTITIONNUM`
+- Use of the `SELECT INTO` statement
+  - Execute `SELECT INTO` in the stand-alone version, the output is a file
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: The output of `SELECT INTO` in the standalone version is a file.

File: docs/en/standalone_vs_cluster.md [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+  - The cluster version of OpenMLDB executes `SELECT INTO`, the output is a directory
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: The output of `SELECT INTO` in the cluster version is a directory.

File: docs/en/standalone_vs_cluster.md [hunk 18]
```diff
@@ -0,0 +1,74 @@
...
+- In the online execution mode of the cluster version, only simple single-table query statements are supported:
+  - Only supports column, expression, and single-row processing functions (Scalar Function) and their combined expression operations
+  - Single table query does not contain [GROUP BY clause](../reference/sql/dql/JOIN_CLAUSE.md), [HAVING clause](../reference/sql/dql/HAVING_CLAUSE.md) and [WINDOW subclause] sentence](../reference/sql/dql/WINDOW_CLAUSE.md)
+  - Single table query only involves the calculation of a single table, and does not design the calculation of multiple tables [JOIN](../reference/sql/dql/JOIN_CLAUSE.md)
```
Comment (lumianph) [inline docs/en/standalone_vs_cluster.md]: - A single table query only involves the computation on a single table, but no [JOIN](../reference/sql/dql/JOIN_CLAUSE.md) based multiple table computation.

**Review by lumianph — CHANGES_REQUESTED** [review]
thanks for the contribution!

File: docs/en/standalone_vs_cluster.md (hunk 18 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 17 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 16 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 15 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 14 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 13 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 12 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 11 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 10 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 9 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 8 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 7 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 6 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 5 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 4 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 3 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 2 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

File: docs/en/standalone_vs_cluster.md (hunk 1 above)
Comment (cswjd) [reply docs/en/standalone_vs_cluster.md]: Ok, thank you.

**Review by lumianph — APPROVED** [review]
LGTM

**Pull request merged**